cmd/kubeadm/app/constants/constants.go

```diff
@@ -426,6 +426,7 @@ var (
 		15: "3.3.10",
 		16: "3.3.15-0",
 		17: "3.3.15-0",
+		18: "3.3.15-0",
```

Comment:
@fabriziopandini  we can remove `12: "3.2.24",` once this merges:
https://github.com/kubernetes/test-infra/pull/14483

Reply:
ok let's wait few days then

Reply:
that PR just merged